README.md

```diff
@@ -36,17 +38,18 @@ Our goals *include*:
     and guessing/experimenting can only get you this close
   * We will not implement useless artificial limitations (max 30 selectable units...)
 * Multiplayer (obviously)
-* Optionally, [Improvements](/doc/ideas/) over the original game
+* Matchmaking and ranking with a [haskell masterserver](https://github.com/SFTtech/openage-masterserver)
+* Optionally, [improvements](/doc/ideas/) over the original game
 * AI scripting in Python, you can use [machine learning](http://scikit-learn.org/stable/)
 * Re-creating [free game assets](https://github.com/SFTtech/openage-data)
 * An easily-moddable content format: **nyan**'s yet another notation
 * A powerful integrated Python console and interface, comparable to [blender](http://blender.org/)
 
-Our goals specifically *exclude*:
```

Comment:
The thing is i wanted to formulate the actual content in a negative way, so it's clear even when read standalone. One newssite even misinterpreted it as "yes we'll do network compat"..
That's why I wanted to clarify it.